errdefs/errors.go

```diff
@@ -43,6 +43,7 @@ var (
 	ErrFailedPrecondition = errors.New("failed precondition")
 	ErrUnavailable        = errors.New("unavailable")
 	ErrNotImplemented     = errors.New("not implemented") // represents not supported and unimplemented
+	ErrWrongPassword      = errors.New("wrong password")
```

Comment:
We have attempted to mirror this error condition with the grpc error codes. As the comment above says, we see those error codes as being very well defined. See https://github.com/grpc/grpc-go/blob/master/codes/codes.go

Reply:
Ah ok - it seems like the most appropriate error code for our use here is `ErrInvalidArgument` by grpc convention, so we can remove this!

Reply:
I removed this error definition now in the latest patch.